src/views/clusteroverview/OverviewTab/getting-started-card/utils/getting-started-grid/GettingStartedGrid.scss

```diff
@@ -44,4 +43,17 @@
       padding-right: var(--pf-v5-c-card--child--PaddingRight);
     }
   }
+
+  &__tooltip {
+    inset: auto auto 0px 5px !important;
```

Comment:
Did you try increasing specificity before adding `!important `?

Reply:
Yes, I did.

Reply:
@pcbailey @metalice, this is the rare occasion. Tried anything to avoid using `!important` (of course!!!), but nothing worked as expected. The problem lies in how PF components' styles are implemented (not well for our UI designs) and how we constantly need to override them and adjust many things around.

Please, provide more specific suggestions or try something, too, otherwise this is not very helpful. Maybe you'll find some way I haven't found. I'd be happy to learn something or to get help with this.

Note that I must have used `inset` together with `!important` to override exactly this prop and its values to achieve the expected look. Any other position-like prop set by me here didn't work, it was overridden at the end.

Thanks.

Reply:
@hstastna , u should better explore all the options before asking us to do it for u.
the issue u are facing caused by __tooltip-icon that is using padding instead of margin, simply change to
&__tooltip-icon {
    margin-left: var(--pf-global--spacer--sm);
  }
and you can remove the inset which is complelty not needed.
BTW u have a bug, when clicking on '?' the tooltip also open the expand section which is unwanted beahiver. thinking popover is more fit for '?' as its expecting a click while tooltip is simply hover. across our app when we have '?' its a popover with a click.

Before:
![image](https://github.com/kubevirt-ui/kubevirt-plugin/assets/14824964/75d06f40-3dc6-46c7-a0c9-2c573a656dca)

After:
![image](https://github.com/kubevirt-ui/kubevirt-plugin/assets/14824964/a95d1d34-bdbe-415c-a618-6d1ef5b1d359)

Reply:
@metalice I am sorry I asked for help before exploring all the options (while there can be so many options for exploring for many days... the new unseen option to solve something can be found anytime). I'll never do such a mistake again. Next time I'll better be stuck for weeks before asking for help.